deploy/helm-charts/server/templates/service.yaml

```diff
@@ -4,7 +4,6 @@ metadata:
   name: {{ .Values.name }}
   namespace: {{ .Release.Namespace }}
 spec:
-  type: LoadBalancer
```

Comment:
No need to create `LoadBalancer` service, until we are using this service from outside of this cluster. It's the user choice to change it afterwards to enhance its usecases.